_posts/2024-07-23-WildFly33-Released.adoc

```diff
@@ -0,0 +1,248 @@
+
+Our recommendation of SE 21 over earlier LTS releases is solely because as a general principle we recommend being on later LTS releases, not because of any problems with WildFly on SE 17 or SE 11.
+
+One reason to use later SE versions is because it gets you ahead of the curve as WildFly and other projects begin to move on from supporting older SE releases.
+
+However, WildFly Preview no longer supports SE 11, as the baseline for Jakarta EE 11 is SE 17. We're continuing to evaluate our plans around SE 11 support in standard WildFly, and I'll be sure to post here as we make decisions. I do encourage WildFly users to prepare now for any eventual change to move off of SE 11.
+
+While we recommend using an LTS JDK release, I do believe WildFly runs well on JDK 22. By runs well, I mean the main WildFly testsuite runs with no more than a few failures in areas not expected to be commonly used. We want developers who are trying to evaluate what a newer JVM means for their applications to be able to look to WildFly as a useful development platform.
+
+Please note that WildFly runs in classpath mode.
+
+== Incompatible Changes
+
+Running WildFly Preview 33 with the Java SecurityManager enabled is no longer allowed. In WildFly Preview 32 it was noted as being unsupported, but with 33 https://issues.redhat.com/browse/WFLY-19287[booting a WildFly Preview server will fail if the SecurityManager is enabled]. This is because the Jakarta EE 11 specifications no longer support the SecurityManager, and it is not possible for WildFly to provide a consistent, reliable SecurityManager experience once the spec APIs and implementation projects we integrate no longer support it.
+
+As noted above, WildFly Preview no longer supports running on Java SE 11.
+
+== Known Issues
+
+The Arquillian version specified in the WildFly 33 end user https://repository.jboss.org/org/wildfly/bom/wildfly-ee-with-tools/33.0.0.Final/wildfly-ee-with-tools-33.0.0.Final.pom[`wildfly-ee-with-tools` BOM] results in https://issues.redhat.com/browse/WFLY-19574[test failures with JUnit 5]. Users can work around this problem by directly controlling the Arquillian version and using Arquillian 1.9.0.
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.jboss.arquillian.container</groupId>
+    <artifactId>arquillian-container-test-spi</artifactId>
+    <version>1.9.0</version>
+</dependency>
+<dependency>
+    <groupId>org.jboss.arquillian.junit</groupId>
+    <artifactId>arquillian-junit-container</artifactId>
+    <version>1.9.0</version>
+    <scope>test</scope>
+</dependency>
+<dependency>
+    <groupId>org.jboss.arquillian.protocol</groupId>
+    <artifactId>arquillian-protocol-servlet-jakarta</artifactId>
+    <version>1.9.0</version>
+</dependency>
+<dependency>
+    <groupId>org.jboss.arquillian.junit5</groupId>
+    <artifactId>arquillian-junit5-container</artifactId>
+    <version>1.9.0</version>
+</dependency>
```

Comment:
Minor nit, these should all probably have `<scope>test</scope>`.

Reply:
Definitely. Fixed.